partner_contact_in_several_companies/README.rst

```diff
@@ -1,8 +1,10 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
-Module name
-===========
+====================================
+Partner Contact in Several Companies
+====================================
 
 This module was written to extend the contact management functionality. It
```

Comment:
I agree that wording sounds better.  Perhaps the template at https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst should be modified as the wording was copied from there.